create/new_project/demos/erc20_btc/src/index.ts

```diff
@@ -173,3 +173,10 @@ async function printBalances(actor: Actor) {
 function toNominal(tokenWei: string, decimals: number) {
     return new BigNumber(tokenWei).div(new BigNumber(10).pow(decimals));
 }
+
+function wait_for_confirmation(text: string) {
+    if (process.env.NON_INTERACTIVE && process.env.NON_INTERACTIVE === "true") {
```

Comment:
```suggestion
    if (process.env.NON_INTERACTIVE) {
```
If you just use `1` or `0` as a value then that should be enough.

Reply:
unfortunately not. The value will be interpreted as a string.